modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/index.es.js

```diff
@@ -12,6 +12,7 @@
  * details.
  */
 
+import HandleSectionAdded from '../js/components/LayoutProvider/handlers/sectionAddedHandler.es';
```

Comment:
I see no use for the `sectionAddedHandler.es` into the `dynamic-data-mapping-form-builder`, so you can move it to the `data-engine/data-engine-js-components-web` and export at its `index.es.js`. Since you are moving the file, take that chance to remove the suffix `.es` from its name.

Reply:
Hi @miltonmc, I made the changes you suggested, I would like you to take a look and let me know if you have anything that needs to be changed.